File: python/K'thLargestElementInArray.py
def largest(arr, n, k): 

	arr.sort(reverse = True) 
	return arr[k-1] 
 
arr = [1,2002,12,6,9,30,10,30] 
n = len(arr) 
k = 3
print(largest(arr, n, k)) 

File: java/K'thLargestElementInArray.java
import java.util.Arrays; 
import java.util.Collections; 

public class Algorithms { 
	public static int largest(Integer[] arr, int k) { 
		Arrays.sort(arr,Collections.reverseOrder()); 
		return arr[k - 1]; 
	} 
	public static void main(String[] args) { 
		Integer arr[] = new Integer[] { 1,2002,12,6,9,30,10,30 }; 
		int k = 3; 
		System.out.print(largest(arr, k)); 
	} 
} 

